Lib/fontgoggles/mac/makePathFromOutline.py

```diff
@@ -1,18 +1,16 @@
 import ctypes
 import pathlib
 
-import freetype
-try:
-    import numpy
-except ImportError:
-    # It's ok to not have numpy if makePathFromArrays() is not used.
-    numpy = None
+import numpy
```

Comment:
I would love to have the soft import of numpy restored: this was relevant in another project which used the fontgoggles lib, but only for OTF, and having numpy as a dependency was both cumbersome and unneccesary.

Reply:
Would it be simpler then to import numby in the function that uses it?

Reply:
Yes, let's do that. I probably did it this way because of some unfounded fear of import overhead: it's cached anyway.